google-datacatalog-kafka-connector/tests/google/datacatalog_connectors/kafka/prepare/datacatalog_entry_factory_test.py

```diff
@@ -45,3 +44,20 @@ def test_topic_metadata_should_be_converted_to_datacatalog_entries(
                 DataCatalogEntryFactoryTestCase.__METADATA_SERVER_HOST,
                 entry.linked_resource)
             self.assertIn(topic, entry.display_name)
+
+    def test_cluster_metadata_should_be_converted_to_dc_entries(
+            self, entry_path):
+        entry_path.return_value = \
+            DataCatalogEntryFactoryTestCase.__MOCKED_ENTRY_PATH
+        metadata = utils.Utils.convert_json_to_object(self.__MODULE_PATH,
+                                                      'test_metadata.json')
+        cluster_id = metadata[MetadataDictKeys.CLUSTER_ID]
+        entry_id, entry = self.__entry_factory.make_entry_for_cluster(metadata)
+        self.assertIsNotNone(entry_id)
```

Comment:
can we compare whole entry to what we expect with one assertion?

Reply:
done :)